examples/parallel_for/seismic/main.cpp

```diff
@@ -91,11 +99,18 @@ int main(int argc, char *argv[]) {
             std::cout << "Substituting 1000 for unlimited frames because not running interactively"
                       << "\n";
         }
+        // TODO : Extend utility::cli_argument_pack() to allow specifying the default value.
+        if (options.numberofIterations == 0) {
```

Comment:
Make it a bit more reliable?
```suggestion
        if (options.numberofIterations <= 0) {
```